### PR TITLE
[Tech] Landing articles redesign

### PR DESCRIPTION
**Pull request by bhlynsky**

### CODE REVIEW

File: docs/website/views/_includes/landing/article-section.njk [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
-              <li class="article-item-taglist-tag">#{{ tag }}</li>
-            {% endfor %}
-          </ul>
-          <p class="article-content fw-300 fs-1 mb-1">{{ article.content }}</p>
-          <span class="article-item-date">{{ article.date }}</span>
-        </a>
-      </li>
-    {% endfor %}
-  </ul>
+  <div class="article-container">
```
Comment (abarmina) [inline docs/website/views/_includes/landing/article-section.njk:19]: in my understanding `container` should also include a title
```suggestion
  <div class="article-content">
```

File: docs/website/views/_includes/landing/article-section.njk [hunk 2]
```diff
@@ -1,31 +1,51 @@
...
+    <div class="article-first article-item">
+      <a class="article-link" href="{{ firstArticle.link }}" target="_blank">
+        <h5 class="article-item-title mb-1">{{ firstArticle.title }}</h5>
```
Comment (abarmina) [inline docs/website/views/_includes/landing/article-section.njk:22]:
```suggestion
        <h5 class="article-item-title mb-2">{{ firstArticle.title }}</h5>
```

File: docs/website/views/_includes/landing/article-section.njk [hunk 3]
```diff
@@ -1,31 +1,51 @@
...
+        <div class="article-first-img mb-1">
```
Comment (abarmina) [inline docs/website/views/_includes/landing/article-section.njk:23]: I think we need to make an image size limit (width, height)

**Review by abarmina — COMMENTED** [review]
what if we'll use one template for all articles using first element check in HTML and grids in LESS? then we will not need {% macro %} and the layout will be clearer

File: docs/website/src/landing/article-section/article-section.less [hunk 4]
```diff
@@ -48,74 +50,80 @@
...
+      text-align: start;
+    }
+
+    .inner-image {
+      min-width: 150px;
+      height: 380px;
+      margin-bottom: 10px;
+    }
+
+    &-img {
```
Comment (abarmina) [inline docs/website/src/landing/article-section/article-section.less:97]: if we make the date separately for all articles after the picture, then we will not need these styles for `.article-item-img` class

Comment (bhlynsky) [reply docs/website/src/landing/article-section/article-section.less:91]: this class is now used to override esl-image's default height (100%), which messes up layout a bit

**Review by abarmina — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Review by Bogdan9817 — APPROVED** [review]

**Pull request merged**